pandas/tests/groupby/test_groupby.py

```diff
@@ -1736,3 +1737,67 @@ def test_groupby_multiindex_series_keys_len_equal_group_axis():
     expected = pd.Series([3], index=ei)
 
     assert_series_equal(result, expected)
+
+
+def test_groupby_observed():
```

Comment:
Can you move these tests to `test_categorical.py`?

Reply:
Done, I also renamed them a bit to fit the other ones in that file better.